Driver/bin/F_Back.in

```diff
@@ -28,6 +28,8 @@ else
     OMNI_JAR2="../XcodeML-Common/build/om-common.jar"
 fi
 
+if [ -z ${OMNI_JAVA+x} ]; then
```

Comment:
This variable stores runtime path to java. Previously it was hard-coded by "make install", now it is possible to set it before running "F_Back". This is for compatibility with spack, where the java from spack is supposed to be picked up instead of system java.